neo/rawio/intanrawio.py

```diff
@@ -81,10 +83,13 @@ class IntanRawIO(BaseRawIO):
     extensions = ["rhd", "rhs", "dat"]
     rawmode = "one-file"
 
-    def __init__(self, filename=""):
+    def __init__(self, filename="", load_data_unsafely=True):
 
         BaseRawIO.__init__(self)
         self.filename = filename
+        self.load_data_unsafely = load_data_unsafely
+        self.unsafe_timestamps = False
```

Comment:
That works for me.

Reply:
I just noticed the default for `load_data_unsafely` is wrong.

Reply:
Yes. Agree. Changing.

Reply:
Done.